website/lib/changelog/generate-component-changelog-entries.mjs

```diff
@@ -93,85 +55,91 @@ const extractVersion = (changelogContent, version) => {
   return match ? match[0] : null;
 };
 
-const convertComponentNameFormat = (componentName) => {
-  const twoLevelComponentNames = ['copy', 'form', 'layout', 'link', 'stepper'];
-  const threeLevelComponentNames = [
-    'stepper-step-indicator',
-    'stepper-task-indicator',
-  ];
-  if (
-    twoLevelComponentNames.includes(componentName.split('-')[0]) &&
-    componentName != 'form'
-  ) {
-    let words = componentName
-      .split('-')
-      .map((word) => word.charAt(0).toUpperCase() + word.slice(1));
-    if (threeLevelComponentNames.includes(componentName)) {
-      return words.join('::');
-    } else {
-      return words[0] + '::' + words.slice(1).join('');
-    }
-  } else {
-    return componentName
-      .split('-')
-      .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
-      .join('');
-  }
-};
-
 const extractComponentChangelogEntries = (components, lastVersionContent) => {
   const componentChangelogEntries = {};
 
-  Object.keys(components).forEach((componentName) => {
-    const formattedComponentName = convertComponentNameFormat(componentName);
-    const regex = new RegExp(`^\`${formattedComponentName}\` - .*$`, 'gm');
+  components.forEach((componentName) => {
```

Comment:
consider renaming this from `componentName` to `componentPath` (more correct, _I think_)